api/Window.json

```diff
@@ -4545,22 +4545,22 @@
               "version_added": true
             },
             "ie": {
-              "version_added": null
+              "version_added": "9"
```

Comment:
Since you suspect that the feature may have been introduced even earlier, let's go with `true` here in stead